src/core/BufferAttribute.js

```diff
@@ -360,7 +380,8 @@ class BufferAttribute {
 
 		if ( this.name !== '' ) data.name = this.name;
 		if ( this.usage !== StaticDrawUsage ) data.usage = this.usage;
-		if ( this.updateRange.offset !== 0 || this.updateRange.count !== - 1 ) data.updateRange = this.updateRange;
+		if ( this._updateRange.offset !== 0 || this._updateRange.count !== - 1 ) data.updateRange = this._updateRange;
```

Comment:
Is this needed if we're aiming for backwards compatibility for 10 releases?

Reply:
Yes, let's keep this until `updateRange` is completely removed.

Reply:
Can't think of a use case when this is needed in the json though...
Maybe we can skip adding `this.updateRanges` this time?

Reply:
Good point. Update range(s) are reset after the render has processed the frame so it's not a typical data that you serialize for a scene graph structure.

Reply:
Great - I removed toJSON and BufferGeometryLoader support for both the old updateRange and updateRanges